themes/bi-connector/theme.conf

```diff
@@ -21,3 +21,4 @@ stable = STABLE
 sitename = MongoDB
 nav_excluded = NAV
 is_upcoming =
+base_url = https://docs.mongodb.com
```

Comment:
Adding this as the default `base_url` to prevent breaking staging/deploying of docs if they don't have `base_url` defined in their `html_theme_options`